crates/shared/src/sources/balancer_v2/pools/weighted_2token.rs

```diff
@@ -0,0 +1,10 @@
+//! Module implementing two-token weighted pool specific indexing logic.
```

Comment:
I guess we don't need to redefine pool info here since we can reuse the one from weighted? Makes me wonder if we really want to have this additional file. Could we just put the content here into weighted or is this a letter of cleanliness?

Reply:
I put it in because we still need a separate contract for indexing events, and will eventually be instantiated with `BalancerWeightedPool2TokenFactory` instead of `BalancerWeighedPoolFactory`. It will probably always stay fairly empty.